File: game_bizModule/src/main/java/com/game/bizModule/human/msg/GCQueryHumanEntryList.java
package com.game.bizModule.human.msg;

import com.game.bizModule.global.AllMsgSerialUId;
import com.game.gameServer.msg.AbstractGCMsgObj;
import com.game.part.msg.type.MsgArrayList;

/**
 * 查询角色入口列表
 *
 * @author hjj2017
 * @since 2015/7/12
 *
 */
public class GCQueryHumanEntryList extends AbstractGCMsgObj {
    /** 玩家角色入口列表 */
    public MsgArrayList<HumanEntryMO> _humanEntryList = new MsgArrayList<>(() -> {
        return new HumanEntryMO();
    });

    @Override
    public short getSerialUId() {
        return AllMsgSerialUId.GC_QUERY_HUMAN_ENTRY_LIST;
    }
}

File: client_wpf/Xgame.GameBizModule/Human/Msg/GCQueryHumanEntryList.cs
using System;

using Xgame.GameClient.Global;
using Xgame.GameClient.Msg;
using Xgame.GamePart.Msg.Type;

namespace Xgame.GameClient.Human.Msg
{
    /// <summary>
    /// 查询角色入口列表
    /// </summary>
    public partial class GCQueryHumanEntryList : BaseGCMsg
    {
        /** 角色入口列表 */
        public MsgArrayList<HumanEntryMO> _humanEntryList;

        // @Override
        public override short MsgSerialUId
        {
            get
            {
                return AllMsgSerialUId.GC_QUERY_HUMAN_ENTRY_LIST;
            }
        }
    }
}
